scripts/requirements

```diff
@@ -4,5 +4,6 @@ REPO_ROOT="$(cd "$(dirname "$0")/.."; pwd)"
 
 export PYTHONPATH="$REPO_ROOT/external/katana/scripts:$REPO_ROOT/scripts:${PYTHONPATH:+:${PYTHONPATH}}"
 python3 -m katana_requirements "$@" || \
-  echo "HINT: If there are ImportErrors make sure you have the yaml and packaging python packages" \
-        "installed. (Apt packages: python3-yaml, python3-packaging)"
+  (echo "HINT:   If there are ImportErrors make sure you have the yaml and packaging python packages" \
```

Comment:
This hint is printed for all non-zero exit values, which is a little confusing.

It seems just as easy to make a dummy module that tries to `import yaml` and prints this hint when there is an actual `ImportError` than to ask users to scan lines of errors to known when to disregard this hint.